packages/workbox-build/test/node/get-file-manifest-entries.js

```diff
@@ -263,38 +263,38 @@ describe('Test getFileManifestEntries', function() {
       globDirectory: path.join(__dirname, '..', '..', '..',
         'workbox-cli', 'test', 'static', 'example-project-1'),
       dynamicUrlToDependencies: {
-        '/template/url1': ['page-1.html', 'index.html'],
```

Comment:
Did some digging around -

- _Workbox-cli  with the bug_ 
 keeps a slash for all assets including template url. This is a problem if the application is not deployed on the root context.

- _Workbox-cli with current fix_ 
 does not put a slash for any assets including template url. This is will work for assets even when not deployed on root context. Template url is something we need to see.

- _sw-precache_

 This does not put a slash to any asset but keeps it for template urls. Even when passed a stripprefix '/' it still keeps the slash :-)

It seems the code for template url and all assets goes thru the same function which removes/adds the leading slash. If we want to keep /, then we would need to split the code. 
In that case should we keep the functionality similar to _sw-precache_ where _stripPrefix_ is not honored for templateurl.